atm.js

```diff
@@ -1,21 +1,95 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+// Checking account - deposit function
+  $(".depositButton").on("click",function(){
+    var deposit  = parseInt($(".depositInput").val())
+    var balance =$(".balance.checking").text()
+    balance = parseInt(balance.replace("$",""))
+    var newBalance = deposit + balance
+    $(".balance.checking").text("$"+ newBalance)
```

Comment:
nice! Try assigning jquery selectors to variables to reduce the amount of Dom querying